workos/types/user_management/oauth_credentials.py

```diff
@@ -0,0 +1,19 @@
+from typing import Literal, List
+from workos.types.workos_model import WorkOSModel
+
+OauthCredentialProviders = Literal[
```

Comment:
Very nit naming suggestion for type
```suggestion
OauthCredentialProviderType = Literal[
```